angular-ngrx-scss/src/app/auth/auth.component.scss

```diff
@@ -1,16 +1,16 @@
 @use '../shared/styles/variables';
 
 .page {
-  width: 100%;
-  height: 100%;
+  width: 100vw;
+  height: 100vh;
```

Comment:
I remember running into issues with this with scrollbars on non-macs. This change used to be present but was removed. What issue is this trying to fix?

Reply:
Oh I just missed a bullet point and see why this was changed. I would figure out why the parent container isn't max width of the page instead. Setting the viewport width like this is not ideal.

Reply:
Ah that's a good point I didn't know about (the previous scrollbars issue). Will move this off into it's own ticket so can do more analysis of it.